templates/github/.github/workflows/scripts/before_install.sh.j2

```diff
@@ -62,7 +62,6 @@ fi
 
 if [[ "$TEST" = "lowerbounds" ]]; then
   python3 .ci/scripts/calc_deps_lowerbounds.py > lowerbounds_requirements.txt
-  mv lowerbounds_requirements.txt requirements.txt
```

Comment:
I love this part of the change!
Shall we name the file `lowerbounds_contstraints.txt` instead?